github-actions/summarize-errors/src/index.ts

```diff
@@ -1,14 +1,28 @@
 import core = require("@actions/core");
 import {ClassDescriptor} from './types';
-import {summarizeErrors} from "./summarizeJUnitErrors";
+import {summarizeErrors as summarizeJUnitErrors} from './summarizeJUnitErrors';
+import {summarizeErrors as summarizeMochaErrors} from './summarizeMochaErrors';
 
 const UNKNOWN_FAILURE_MESSAGE = `Something failed with this project, but the action can't tell what. Consult the artifact and check manually.
 Make sure you check code coverage!`;
 
+const SUMMARIZER_BY_FRAMEWORK = new Map([
```

Comment:
If we need to support more frameworks in the future (for whatever reason), we can do that by adding to this.